package.json

```diff
@@ -48,19 +49,19 @@
     "react": "16.8.6",
     "react-dom": "16.8.6",
     "rimraf": "2.6.3",
-    "semantic-release": "15.13.12",
+    "semantic-release": "15.13.27",
     "ts-jest": "24.0.2",
     "typescript": "3.4.5"
   },
   "dependencies": {
-    "dompurify": "1.0.10"
+    "dompurify": "2.0.7"
   },
   "peerDependencies": {
     "react": "^16.2.0"
   },
   "husky": {
     "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS --extends @commitlint/config-conventional"
```

Comment:
`@commitlint/config-conventional` is not the same style. If you put that `extends` you can remove the `.commitlintrc` file and use the new convention